ranges.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Semantic Version Ranges
+---
+
+Semantic Version Range Selectors
+================================
+
+Summary
+-------
+
+A SemVer `Range` is a set of `Comparators` that specify a set of SemVer
+Versions.
+
+A Comparator is composed of an `Operator` and a `Version`.  The set
+of primitive Operators is:
+
+* `<` Less than
+* `<=` Less than or equal to
+* `>` Greater than
+* `>=` Greater than or equal to
+* `=` Equal.  If no operator is specified, then equality is assumed,
+  so this operator is optional, but MAY be included to differentiate it
+  from a SemVer Version string.
```

Comment:
This is one area where Cargo differs from Node, and so I'd prefer this latter part be relegated to elsewhere where we talk about implementation defined things.